libs/partners/mongodb/pyproject.toml

```
@@ -4,7 +4,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "langchain-mongodb"
-version = "0.1.7"
+version = "0.2.0"
```

```suggestion
version = "0.1.8"
```
This is not a breaking change right?

---

Prakul's convinced me that making a breaking change for a small improvement in UX isn't worth it, so I'll only bump to 0.1.8 and roll back type of pre_filters. Doing this afternoon or tomorrow morning.